src/app/root-store/server-data/user-tags/user-tags.selectors.ts

```diff
@@ -113,3 +113,8 @@ export const getServiceOfferingParamTags = createSelector(
     return tags.filter(tag => tag.key.startsWith(userTagKeys.computeOfferingParam));
   }
 );
+
+export const getKeyboardLayout = createSelector(
+  getUserTagsEntities,
+  (entities): string => entities[userTagKeys.keyboardLayoutForVms] && entities[userTagKeys.keyboardLayoutForVms].value
```

Comment:
Let's use `(entities): string => entities[userTagKeys.keyboardLayoutForVms].value` without check. It can be confising why just only this selector has this check? Can it be undefined unlike the rest?